### DISCUSSION TITLE
Azure B2C on MAUI app with Azure Data Sync Offline

### DISCUSSION THREAD

**Discussion by StargazerNC** [Azure/azure-mobile-apps#713] [Q&A]
Hello everyone,

We're developing a new app and we are integrating Azure B2C authentication on it. I was wondering if there was any fairly thorough sample or walkthrough. 

My main objective is to check if our implementation is following the recommended path. 

Thanks!

**Comment by adrianhall** [comment]
AAD B2C uses MSAL as its core, so you can use a variant of the GetTokenAsync() method that we use in the TodoApp demo app for this.
